src/NServiceBus.Core/Routing/DistributionPolicy.cs

```diff
@@ -13,10 +13,11 @@ public void SetDistributionStrategy(string endpointName, DistributionStrategy di
         public DistributionStrategy GetDistributionStrategy(string endpointName)
         {
             DistributionStrategy configuredStrategy;
-            return configuredStrategies.TryGetValue(endpointName, out configuredStrategy) ? configuredStrategy : defaultStrategy;
+            return configuredStrategies.TryGetValue(endpointName, out configuredStrategy) 
```

Comment:
I think this might be better expressed by
```csharp
return configuredStrategies.GetOrAdd(endpointName, key => new SingleInstanceRoundRobinDistributionStrategy());
```

I chose the overload the accepts a `Func<TKey, TValue>` since that version won't allocate when then key exists in the collection.